lighthouse-cli/test/smokehouse/dobetterweb/dbw-expectations.js

```diff
@@ -16,6 +16,64 @@ module.exports = [
       }, {
         id: 'wordpress',
       }],
+      TagsBlockingFirstPaint: [
```

Comment:
the user-observable behavior here is in the `render-blocking-resources` audit (though `TagsBlockingFirstPaint` will enter our `PublicGathererArtifacts` someday).

There is a balance to strike when testing internal state when it's difficult to test otherwise, so I understand the argument for capturing it here, but if we accept that *and* that this data is important to assert, then we should assert it in the audit as well.

Reply:
would you prefer I change the alternate request to a large stylesheet so we can assert in the audit too?

Reply:
> would you prefer I change the alternate request to a large stylesheet so we can assert in the audit too?

that might make the alternate stylesheet test more realistic, but not being included there is also a signal that the audit is working as expected

Reply:
sorry, but what's the requested change here?

Reply:
> sorry, but what's the requested change here?

if whatever items you don't want to see regress (e.g. the urls, etc) are important enough to add to a smoke test, they should be asserted where they are actually observable by a user? The artifact can be asserted all day but that doesn't matter if the audit is mismatched to it.

Reply:
talked offline - didn't realize his suggestions are independent of the specific alternate-stylesheet I added. should expand the `render-blocking-resources` audit expectation to be more than "`detail.items` is array of 7 items".

Reply:
yeah, sorry, we were having two different conversations :)